canonical-websites/www.ubuntu.com
# Update archived Japanese service description date. Add redirect. Add notice to top of the page.

## Done

Updated the Japanese service description with a notice and moved the archived one to the correct date.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-06-27) and see that you are redirected.
- Go to [http://0.0.0.0:8001/](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja)
  - See that the older versions link has been updated
  - See that three lines have been added under the title.
